GitUI/UserControls/RevisionGrid/FilterInfo.cs

```diff
@@ -80,6 +80,7 @@ public string DiffContent
 
         public int CommitsLimitDefault => AppSettings.MaxRevisionGraphCommits;
 
+        [System.Diagnostics.DebuggerBrowsable(DebuggerBrowsableState.Never)]
```

Comment:
What prompted this addition?

Reply:
Msvs could not inspect when debugging FilterInfo, stopped with exception.

Reply:
What exception did you get? Can you share repro steps?